masterlist.yaml

```diff
@@ -8450,26 +8450,30 @@ plugins:
   - name: 'Town Guard Armor Replacers.esp'
     url: [ 'http://tesalliance.org/forums/index.php?/files/file/563-town-guard-armor-replacer' ]
     tag: [ Graphics ]
-  - name: 'mz_Shields.esp'
+
+  - name: 'mz_Shields(_SI)?\.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/10376' ]
-    tag:
-      - Deactivate
-      - Graphics
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Magic Shields' ]
+        condition: 'many("mz_Shields(_SI)?\.esp")'
+    tag: [ Graphics ]
+
```

Comment:
@Infernio For consistent formatting, entries for plugins on the same page don't need to be separated by a newline. Just entries for plugins on different pages.

```suggestion
```